engines/docker/generated_configschema.go

```diff
@@ -0,0 +1,47 @@
+// This source code file is AUTO-GENERATED by github.com/taskcluster/jsonschema2go
+
+package docker
+
+import "github.com/taskcluster/taskcluster-worker/runtime"
+
+type (
+	// Config applicable to docker engine
+	Config struct {
+
+		// Root Volume blah blah
+		RootVolume string `json:"rootVolume"`
+	}
+)
+
+func ConfigSchema() runtime.CompositeSchema {
```

Comment:
> Notice the subtle }() that the end...

D'oh! Yeah, totally makes sense. This also has the nice side effect that `init()` functions inside plugin/engine packages that register plugin and engine providers can even safely refer to the generated variables (since variables are initialised before `init()` is called). :+1: